wpiutil/src/main/java/edu/wpi/first/util/protobuf/ProtobufSerializable.java

```diff
@@ -10,6 +10,7 @@
  * Marker interface to indicate a class is serializable using Protobuf serialization.
  *
  * <p>While this cannot be enforced by the interface, any class implementing this interface should
- * provide a public final static `proto` member variable.
+ * provide a public final static `proto` member variable, or a static final `getProto()` method if
+ * the class is generic.
```

Comment:
Unrelated change?

Reply:
This PR introduces templated protobuf instantiations for the first time, and I wanted to update the doc comment's suggestion to accommodate non-templated and templated protobufs.